packages/scandipwa/src/component/SearchOverlay/SearchOverlay.style.scss

```diff
@@ -15,31 +15,85 @@
     --search-criteria-search-term-color: #0a0a0a;
 }
 
+@keyframes slideIn {
```

Comment:
these animations seem to be common with recent UX requirements, let's create mixins for it to make them reusable across the whole project.
can check the similar in this PR: [PERF-25](https://github.com/scandipwa/scandipwa/pull/5533/files)
```
@mixin collapse-content {
    transition: max-height 0s ease;
    animation: collapse-content var(--content-expansion-duration) ease forwards;

    @keyframes collapse-content {
        from {
            max-height: 300vh;
            opacity: 1;
        }

        to {
            max-height: 0;
            padding-block: 0;
            opacity: 0;
        }
    }
}
```